.github/workflows/publish.yml

```
@@ -0,0 +1,34 @@
+---
+name: publish
+
+on:
+  push:
+    tags:
+      - v*
+permissions:
+  contents: write
+env:
+  GRADLE_SWITCHES: --console=plain --info --warning-mode=all
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Get the TAG version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+      - uses: gradle/wrapper-validation-action@v1
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 19
```

Optional, but would 17 work here?
```suggestion
          java-version: 17
```